test/performance/specs/front-end-block-theme.spec.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
```

Comment:
`@wordpress/e2e-test-utils-playwright` is designed to be used in e2e testing environment so it might contain some optimizations or setup to skew the performance metrics. I think it's better to use the vanilla API from `@playwright/test` directly.

Reply:
Good point. We also use the e2e utils with Puppeteer, so again, I'd leave that optimization for a follow-up PR. I'll make a list of the follow-ups, as it's getting long! 😅

Reply:
Huh, that setup might indeed be skewing those initial painting numbers. The authentication or the rest utils are pretty useful though, so maybe instead of in a global setup, we'd start each page vanilla and enable utils selectively when needed?

Reply:
The issue really is we shouldn't use `test` directly from the package. The package itself is side-effect-free, but it's not as soon as we use the `test` util. Instead, we can build our own `test` using the primitives the package provides. It's a bit of an advanced usage though and not mentioned in the doc 😅, so feel free to ask questions!